### PR TITLE
Framework

### PR DESCRIPTION
**Pull request by madsailor**
After the great work that harryyoung did organizing the structure, I thought it would be helpful to set up a framework of available science for each .cfg file.  This should make it easier for anyone/everyone to add their science report for the specific situation that they intend it for.  Essentially it is just code commenting, but it helps avoid having to look up whether you can do a certain type of science in a certain situation.  This framework is based on info at http://wiki.kerbalspaceprogram.com/wiki/Science#Science_activities and http://wiki.kerbalspaceprogram.com/wiki/Biome.
I haven't been able to test that MM handles all this '//' commenting as intended yet, so It is definitely worth verifying before merging this.  Also, I fleshed it out only for Eve to get your opinion before continuing.
If you find this a useful direction to take this mod It should be fairly simple, though a bit of a slog, to implement it for the rest of the planets and moons.  But once the work is done it should make adding content a breeze.

### CODE REVIEW

File: planets/eve/explodium sea/barometerScan.cfg [hunk 1]
```diff
@@ -2,6 +2,14 @@
 {
 	%RESULTS
 	{
-		//LAST_ENTRY
+                // Science Data is available in different situations for each
+                // planetary body. Please use this framework as a guide to the
+                // situational science that is available for this body. You may
+                // copy-paste and remove '//' to add your entry. Please add your
+                // entry just before '//LAST_ENTRY' or with similar entries.
+                //EveSrfSplashedExplodiumSea = Your science report text here.
```
Comment (Olympic1) [inline planets/eve/explodium sea/barometerScan.cfg]: You can also land in water so you can add "EveSrfLandedExplodiumSea = Your science report text here."

**Comment by Olympic1** [comment]
You can also land in water so you can add "EveSrfLandedExplodiumSea = Your science report text here." to all the explodium sea reports.

**Comment by madsailor** [comment]
Isn't 'landing in water' what KSP registers as 'SrfSplashed'?  So in this case it's: EveSrfSplashedExplodiumSea = Your science report text here.  This is what I added to all the applicable instruments in the explodium sea folder, and deleted the .cfg files for atmosphereAnalysis and seismicScan since they cannot be used splashed down.

**Comment by Olympic1** [comment]
KerbinSrfLandedWater can be found in the ScienceDefs.cfg from Squad, It works since 0.22. It can be achieved by landing slowly into the water.

**Comment by madsailor** [comment]
Ok. I didn't realize that was possible as it wasn't listed in the wiki, and I had never tried it myself. I can add EveSrfLandedWater to the .cfg's in the explodium sea folder.  I'll have it committed in a few minutes, but again some testing is in order before you consider merging my pull request.....uncharted non-surface-landed waters for me here ;)

**Comment by madsailor** [comment]
Ok, I can verify that these changes do not catastrophically crash MM or KSP.  However, I can also verify that new science reports added to the files I altered are not being used in-game.  Only the reports from the default folder are being used.  I am not sure if this is due to my changes or the re-structuring of the .cfg files into directories.  Unfortunately I will be away for a few days, so I won't be able to further test until then. Feel free to peruse, test, and change if you find a working solution, and I will pick it back up when I return.

**Comment by DuoDex** [comment]
You've removed a couple of lines at the top. Working in your repo right now.

**Comment by madsailor** [comment]
I had been testing landed on water at eve as discussed earlier.  It was a simple mistake - used EveSrfLandedWater instead of EveSrfLandedExplodiumSea.  Changed it and now new science reports showing up as expected.  Feel free to request changes, or wait for me to get back and finish the framework for the rest of the planets, but eve is complete and working as expected.

**Comment by madsailor** [comment]
I finished work on the rest of the planets/moons and merged it back into the master branch of my repo.  There were 3 conflicts from edits in files for KSC, changing , and ' since they were showing up as byte arrays for me in gedit for some reason.  To simplify things I am going to close this pull request and open another one from my already-merged-and-up-to-date master.

**Pull request closed without merging**